### PR TITLE
Feature/signup login with phone

### PR DESCRIPTION
**Pull request by SelimGribaa**
Asana Task : https://app.asana.com/0/1141396442496489/1200239351907878

### CODE REVIEW

**Review by rchapel — COMMENTED** [review]
It will be great to provide a description of the task in the ticket or in the PR description.

File: IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift [hunk 1]
```diff
@@ -116,20 +116,20 @@ public class ReachFive: NSObject {
...
-        }.onFailure { error in
-            var messageAlert = ""
-            switch error {
-            case .RequestError(let requestErrors):
-                messageAlert = requestErrors.errorUserMsg!
-            case .TechnicalError(let technicalError):
-                messageAlert = (technicalError.apiError?.errorUserMsg)! as String
-            default:
-                messageAlert = error.localizedDescription
+            }.onFailure { error in
```
[the remark this answers is not in the document]
Comment (SelimGribaa) [reply IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift:119]: The base of this PR is Master, error handling has been dealt with in a previous PR

File: IdentitySdkCore/IdentitySdkCore/Classes/ReachFiveError.swift [hunk 2]
```diff
@@ -5,6 +5,7 @@ public enum ReachFiveError: Error {
     case AuthFailure(reason: String, apiError: ApiError? = nil)
     case AuthCanceled
     case TechnicalError(reason: String, apiError: ApiError? = nil)
+    case Fido2Error(reason: String)
```
[the remark this answers is not in the document]
Comment (SelimGribaa) [reply IdentitySdkCore/IdentitySdkCore/Classes/ReachFiveError.swift:8]: Done

**Comment by roxanemace** [comment]
I don't understand why there are changes to Webauthn concerning the ticket's scope? 🤔

File: IdentitySdkCore/IdentitySdkCore/Classes/ReachFiveFidoClient.swift [hunk 3]
```diff
@@ -44,33 +44,43 @@ class ReachFiveFidoClient: NSObject
...
-        
         options.attestation = .direct
         options.addPubKeyCredParam(alg: .es256)
+        
+        var arrayPublicKeyCredentialDescriptor = [PublicKeyCredentialDescriptor] ()
+        
+        if (registrationOption.options.publicKey.excludeCredentials != nil)
+        {
+            for r5PublicKeyCredentialDescriptor in registrationOption.options.publicKey.excludeCredentials! {
+                var publicKeyCredentialDescriptor = PublicKeyCredentialDescriptor ()
```
[the remark this answers is not in the document]
Comment (SelimGribaa) [reply IdentitySdkCore/IdentitySdkCore/Classes/ReachFiveFidoClient.swift:72]: Done

**Review by thomasreach5 — APPROVED** [review]

**Pull request merged**